Sprint-3/revise/implement/password-validator.js

```diff
@@ -0,0 +1,34 @@
+function passwordValidation(password) {
+  //   if (existingPasswords.includes(password)) {
```

Comment:
You can implement another function to return an array of existing passwords, and call that function in both `passwordValidation()` and in `password-validator.test.js`.